extensions/machine-learning-services/src/modelManagement/azureModelRegistryService.ts

```diff
@@ -139,15 +153,14 @@ export class AzureModelRegistryService {
 
 		try {
 			for (const tenant of account.properties.tenants) {
-				const tokens = await this._apiWrapper.getSecurityToken(account, azdata.AzureResource.ResourceManagement);
-				const token = tokens[tenant.id].token;
-				const tokenType = tokens[tenant.id].tokenType;
-				const client = new AzureMachineLearningWorkspaces(new TokenCredentials(token, tokenType), subscription.id);
+				const client = await this.getAmlClient(account, subscription, tenant);
 				let result = resourceGroup ? await client.workspaces.listByResourceGroup(resourceGroup.name) : await client.workspaces.listBySubscription();
-				resources.push(...result);
+				if (result) {
+					resources.push(...result);
+				}
 			}
 		} catch (error) {
-
+			throw error;
```

Comment:
Anything else supposed to happen in this catch block, or is it just for debugging?